_specs/ecip-10746.md

```diff
@@ -0,0 +1,96 @@
+---
+ecip: 1076
+title: Mining Algorithm Change Process
+author: Talha Cross (@soc1c)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/174
+status: Draft
+type: ECBP
+created: 2019-11-16
+---
+
+### Abstract
+This meta document is agnostic to any mining algorithm. Its sole purpose is specifying a process how to openly determine and select a mining algorithm for Ethereum Classic.
+
+### Motivation
+The decision to change the Ethereum Classic mining algorithm - or to keep it as is - is not an easy one to make. It should be done by broad consensus with all stakeholders involved - miners, investors, application and core developers, and anyone else who feels having a stake in ETC. This meta document proposes a process of how to facilitate the potential change of the mining algorithm and parameters.
+
+### Available Options
+Currently, the following options are debatable.
+
+- **ECBP-1076-A**: _"Ethash Status Quo."_ The mining algorithm will be untouched as it is since genesis, as **Ethash** function with a statically **increasing DAG**.
+- **ECBP-1076-B**: _"Ethash Limited DAG."_ The mining algorithm will be untouched as it is since genesis, as **Ethash** function. However, the **DAG size will be limited** as specified in [ECIP-1043](https://ecips.ethereumclassic.org/ECIPs/ecip-1043).
+- **ECBP-1076-C**: _"SHA3."_ The mining algorithm will be **changed to SHA3** as specified in [ECIP-1049](https://ecips.ethereumclassic.org/ECIPs/ecip-1049).
```

Comment:
I'd change it to Keccak256 for consistency and more technical accuracy and to reflect the language in ECIP-1049

Reply:
Keccak256 might be confused with the Keccak256 candidate used in Ethereum though.

Reply:
Perhaps then Keccak256 (SHA3)? Maybe @prestwich can weigh in with his thoughts too? Its only to ensure the language communicates the correct engineering implementation.

Reply:
because Ethereum introduced a lot of naming confusion, you might consider defining this by the output it produces for a set of inputs. 

e.g. the hash function that produces the digest `0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470` when given the preimage `0x`,  which we refer to as `keccak256`

Reply:
Yeah agree with @YazzyYaz here -- `keccak256` would be more precise.

Reply:
Done